app/models/manageiq/providers/base_manager/event_catcher/runner.rb

```diff
@@ -20,10 +23,48 @@ def after_initialize
     _log.info "#{log_prefix} Event Catcher skipping the following events:"
     $log.log_hashes(@filtered_events)
 
+    configure_event_flooding_prevention if worker_settings.try(:[], :flooding_monitor_enabled)
```

Comment:
Why do you have to `try` here?
As `worker_settings` is a `Hash`, can't you just access the key?